experiments/clbpips/generate.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python3
 
-import sys, re
-
 from prjxray.segmaker import Segmaker
+from prjxray import util
 
-segmk = Segmaker("design.bits")
+segmk = Segmaker(util.get_db_root(), "design.bits")
```

Comment:
every fuzzer is expected to pass "util.get_db_root()". This should be a settable default

Reply:
What is meant by a "settable default"?  You mean a default argument?

I'm not sure why making the database root implicit to Segmaker is buying us, besides saving some characters.  Environment variable usage should be a decision made by the program, rather than the library, and I think keeping the environment variable usage to as few places is important.

Reply:
Making the code more readable is important to me. Its a very reasonable default. The program still has the option to override if it wants to